Added example for limiting workflow concurrency in .NET fan-in/out example

WhitWaldo opened the pull request:
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
I love rich documentation. It's so satisfying to read something that details a new concept, shares examples and really sets me up to succeed. But I don't like it when said documentation introduces another idea and leaves me to my own devices to figure out how to do it.

Tonight I was reading about the different workflow patterns. In the fan-out/fan-in pattern, it explains how a series of parallel tasks can be completed at once, but then it leaves a note at the end: "While not shown in the example, it's possible to go further and limit the degree of concurrency using simple, language-specific constructs."

As it took me a minute to figure out a worthwhile .NET approach to doing this, I wanted to contribute it back so future me isn't left hanging. I leave it to others to figure out other language equivalents.


## Issue reference

None - saw this in the docs and after determining a solution, wanted to contribute it back.

WhitWaldo committed:
Added example for retrieving bulk deserialized items matching PR #1173 in dotnet-sdk repo.
Signed-off-by: Whit Waldo <whit.waldo@innovian.net>

WhitWaldo committed:
Update daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Whit Waldo <whitson.waldo@gmail.com>

hhunter-ms committed:
Merge pull request #3910 from JoshVanL/preview-feature-component-hot-reloading

Component Hot Reloading

elena-kolevska committed:
Apply suggestions from code review

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Elena Kolevska <elena-kolevska@users.noreply.github.com>

passuied committed:
Feature/add-kafka-pubsub-schema-registry (#3946)

* Added doc for kafka pubsub Avro schema registry support

Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update setup-apache-kafka.md

Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

---------

Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>
Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

passuied committed:
fix formatting (#3950)

Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

msfussell committed:
Merge branch 'v1.13' into error-codes

elena-kolevska committed:
Apply suggestions from code review

Signed-off-by: Elena Kolevska <elena-kolevska@users.noreply.github.com>

elena-kolevska committed:
Small fixes after review

Signed-off-by: Elena Kolevska <elena@kolevska.com>

elena-kolevska committed:
Small update

Signed-off-by: Elena Kolevska <elena@kolevska.com>

hhunter-ms committed:
update howtos and patterns

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
add link to js sdk

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3869' of https://github.com/hhunter-ms/docs into issue_3869

hhunter-ms committed:
Merge branch 'v1.13' into issue_3869

hhunter-ms committed:
fix localized link

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3869' of https://github.com/hhunter-ms/docs into issue_3869

elena-kolevska committed:
Add related links

Signed-off-by: Elena Kolevska <elena@kolevska.com>

hhunter-ms committed:
Merge pull request #3908 from elena-kolevska/error-codes

Error codes

hhunter-ms committed:
tweak for limitations

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3868

hhunter-ms committed:
upmerge

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge pull request #3976 from hhunter-ms/upmerge_01-26c

Upmerge 1.12 --> 1.13

hhunter-ms committed:
prep 1.13 for release

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

kaibocai committed:
Update limitation for js

Update limitation for js

Signed-off-by: kaibocai <89094811+kaibocai@users.noreply.github.com>

kaibocai committed:
Update workflow-features-concepts.md

Signed-off-by: kaibocai <89094811+kaibocai@users.noreply.github.com>

hhunter-ms committed:
add monitor pattern

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3869

WhitWaldo committed:
Added example for limiting workflow concurrency in .NET

Signed-off-by: Whit Waldo <whit.waldo@innovian.net>

WhitWaldo committed:
Added missing close tab markup and surrounding tabs

Signed-off-by: Whit Waldo <whit.waldo@innovian.net>

hhunter-ms committed:
Merge branch 'v1.12' into limit-workflow-concurrency-dotnet

hhunter-ms commented:
Thanks for this contribution @WhitWaldo!

cgillum commented on daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-patterns.md:
```diff
@@ -287,7 +287,65 @@ The key takeaways from this example are:
+    foreach(var task in tasks)
+    {
+      //Wait for the semaphore to become available
+      await semaphore.WaitAsync();
```
This actually violates one of the workflow code constraints, which is that you mustn't await any task that isn't created by the context object, so unfortunately we can't use `SemaphoreSlim` for this. Instead, the right way to do this is to follow an approach similar to what is posted here for Durable Functions: https://stackoverflow.com/a/58794510/2069.

WhitWaldo replied:
Haha, whoops - further illustrating why it's frustrating for the docs to leave this as an exercise to the reader to do properly. I'll review your answer there and adapt it for a second attempt. Thank you!

cgillum replied:
Haha, that's a great point. 😅 Looking forward to the update! This is one of those questions we get a lot and I always kick myself for not having it properly documented somewhere. Thanks for doing this.

kaibocai committed:
Update daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: kaibocai <89094811+kaibocai@users.noreply.github.com>

hhunter-ms committed:
update diagrams for middleware

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

ItalyPaleAle committed:
[1.13] Add docs for SQLite name resolver

Fixes #3914

Signed-off-by: ItalyPaleAle <43508+ItalyPaleAle@users.noreply.github.com>

famarting committed:
add kafka aws iam to docs

Signed-off-by: Fabian Martinez <46371672+famarting@users.noreply.github.com>

hhunter-ms committed:
Merge branch 'v1.13' into patch-2

ItalyPaleAle committed:
Apply suggestions from code review

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

hhunter-ms committed:
Merge pull request #3982 from kaibocai/patch-2

Update limitation for js

hhunter-ms committed:
Merge branch 'v1.13' into issue_3869

Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

ItalyPaleAle committed:
Update docs for low-cardinality metrics

Fixes #3987

Includes documenting the new "http.increasedCardinality" configuration option

Signed-off-by: ItalyPaleAle <43508+ItalyPaleAle@users.noreply.github.com>

msfussell committed:
Merge branch 'v1.13' into sqlite-nr

ItalyPaleAle committed:
Update daprdocs/content/en/reference/components-reference/supported-name-resolution/nr-sqlite.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

ItalyPaleAle committed:
[1.13] Azure Blob Storage v2 docs

Fixes #3203

Signed-off-by: ItalyPaleAle <43508+ItalyPaleAle@users.noreply.github.com>

msfussell committed:
Merge branch 'v1.13' into kafka-aws-iam

ItalyPaleAle committed:
Update daprdocs/content/en/operations/observability/metrics/metrics-overview.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

ItalyPaleAle committed:
Apply suggestions from code review

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

ItalyPaleAle committed:
Updated configuration-schema

Signed-off-by: ItalyPaleAle <43508+ItalyPaleAle@users.noreply.github.com>

ItalyPaleAle committed:
Merge branch 'v1.13' into metrics-cardinality

ItalyPaleAle committed:
[1.13] Document actor services flags

Adds docs for the daprd flags `--actors-service` and `--reminders-service`. These are currently not very useful to end users, as there's only 1 possible implementation in Dapr today.

Fixes #3948

Signed-off-by: ItalyPaleAle <43508+ItalyPaleAle@users.noreply.github.com>

hhunter-ms committed:
update go snippets

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3868' of https://github.com/hhunter-ms/docs into issue_3868

hhunter-ms committed:
Merge branch 'v1.13' into issue_3868

Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

hhunter-ms committed:
add quickstart

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3868' of https://github.com/hhunter-ms/docs into issue_3868

shivamkm07 committed:
Adding recursive terminate/purge in workflow API (#3969)

* Adding recursive terminate/purge in workflow api

Signed-off-by: Shivam Kumar <shivamkm07@gmail.com>

* Apply suggestions from code review

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Shivam Kumar <shivamkm07@gmail.com>

* review comments

Signed-off-by: Shivam Kumar <shivamkm07@gmail.com>

---------

Signed-off-by: Shivam Kumar <shivamkm07@gmail.com>
Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

msfussell committed:
Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Mark Fussell <markfussell@gmail.com>

msfussell committed:
Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Mark Fussell <markfussell@gmail.com>

msfussell committed:
Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Mark Fussell <markfussell@gmail.com>

msfussell committed:
Update daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Mark Fussell <markfussell@gmail.com>

msfussell committed:
Merge branch 'v1.13' into kafka-aws-iam

hhunter-ms committed:
Merge pull request #3992 from famarting/kafka-aws-iam

Add Kafka AWS IAM support for authenication

hhunter-ms committed:
Merge branch 'v1.13' into metrics-cardinality

ItalyPaleAle committed:
Merge branch 'v1.13' into sqlite-nr

msfussell committed:
Update daprdocs/content/en/operations/observability/metrics/metrics-overview.md

Signed-off-by: Mark Fussell <markfussell@gmail.com>

hhunter-ms committed:
Merge pull request #3995 from ItalyPaleAle/metrics-cardinality

[1.13] Update docs for low-cardinality metrics

ItalyPaleAle committed:
Update daprdocs/content/en/reference/components-reference/supported-name-resolution/nr-sqlite.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

msfussell committed:
Update daprdocs/content/en/reference/components-reference/supported-name-resolution/nr-sqlite.md

Signed-off-by: Mark Fussell <markfussell@gmail.com>

msfussell committed:
Update daprdocs/content/en/reference/components-reference/supported-name-resolution/nr-sqlite.md

Co-authored-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>
Signed-off-by: Mark Fussell <markfussell@gmail.com>

msfussell committed:
Update nr-sqlite.md

Fixed formatting

msfussell committed:
Update daprdocs/content/en/reference/components-reference/supported-name-resolution/nr-sqlite.md

Signed-off-by: Mark Fussell <markfussell@gmail.com>

hhunter-ms committed:
Merge branch 'v1.13' into sqlite-nr

ItalyPaleAle committed:
Apply suggestions from code review

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

hhunter-ms committed:
Merge pull request #3991 from ItalyPaleAle/sqlite-nr

[1.13] Add docs for SQLite name resolver

hhunter-ms committed:
Merge branch 'v1.13' into issue_3869

ItalyPaleAle committed:
Merge branch 'v1.13' into actors-services

hhunter-ms committed:
add supported SDKs

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
add name resolution to config overview and schema

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
update titles, add links, update service invo overview

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

ItalyPaleAle committed:
Update daprdocs/content/en/reference/components-reference/supported-state-stores/setup-azure-blobstorage.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

hhunter-ms committed:
mark edit

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3869' of https://github.com/hhunter-ms/docs into issue_3869

hhunter-ms committed:
mike initial review

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3868

hhunter-ms committed:
remove older versions

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

msfussell committed:
Merge branch 'v1.13' into azure-blob-v2

ItalyPaleAle committed:
Update daprdocs/content/en/reference/components-reference/supported-state-stores/setup-azure-blobstorage.md

Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

hhunter-ms committed:
add js limitation

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

JoshVanL committed:
Sentry Token Validator spec

Adds documentation about sentry token validator spec, and control plane trust domain.

Signed-off-by: joshvanl <me@joshvanl.dev>

ItalyPaleAle committed:
[1.13] Docs for PostgreSQL v2 state store (#3996)

* [1.13] Docs for PostgreSQL v2 state store

Fixes #3911

Signed-off-by: ItalyPaleAle <43508+ItalyPaleAle@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

* Update daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v1.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

* Update daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v2.md

Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>

---------

Signed-off-by: ItalyPaleAle <43508+ItalyPaleAle@users.noreply.github.com>
Signed-off-by: Alessandro (Ale) Segala <43508+ItalyPaleAle@users.noreply.github.com>
Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

hhunter-ms committed:
Merge branch 'v1.13' into azure-blob-v2

hhunter-ms committed:
Merge pull request #3997 from ItalyPaleAle/azure-blob-v2

[1.13] Azure Blob Storage v2 docs

hhunter-ms committed:
mark review

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_4005

hhunter-ms committed:
forgot one

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_4005' of https://github.com/hhunter-ms/docs into issue_4005

hhunter-ms committed:
Merge pull request #4007 from hhunter-ms/issue_4005

Update name resolution docs

hhunter-ms committed:
Merge branch 'v1.13' into 1.13_versions_navbar

hhunter-ms committed:
remove nats streaming

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge pull request #4010 from hhunter-ms/1.13_versions_navbar

[Endgame] Remove *old* versions from 1.13 navbar

hhunter-ms committed:
Merge branch 'v1.13' into issue_3998

hhunter-ms committed:
Merge pull request #4019 from hhunter-ms/issue_3998

Remove NATS streaming component (deprecated in 1.13)

hhunter-ms committed:
Merge branch 'v1.13' into actors-services

hhunter-ms committed:
Merge branch 'v1.13' into issue_3869

hhunter-ms committed:
Merge pull request #3896 from hhunter-ms/issue_3869

[Workflow] Update for JavaScript SDK

ASHIQUEMD committed:
Add workflow backend documentation

Signed-off-by: MD Ashique <noorani.ashique5@gmail.com>

JoshVanL committed:
Adds review comments

Signed-off-by: joshvanl <me@joshvanl.dev>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3868

Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

msfussell committed:
Merge branch 'v1.13' into actors-services

msfussell committed:
Merge branch 'v1.13' into sentry-token-validator

JoshVanL committed:
Workflow terminate/purge: remove `non_recursive` query option (#4008)

* Updates workflow terminate/purge `non_recursive` query to `recursive`

Updates from PR https://github.com/dapr/dapr/pull/7498

Signed-off-by: joshvanl <me@joshvanl.dev>

* Removes recursive workflow delete strategy from API completely

Signed-off-by: joshvanl <me@joshvanl.dev>

---------

Signed-off-by: joshvanl <me@joshvanl.dev>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

msfussell committed:
Merge branch 'v1.13' into actors-services

hhunter-ms committed:
Merge branch 'v1.13' into sentry-token-validator

hhunter-ms committed:
Merge pull request #4017 from JoshVanL/sentry-token-validator

Sentry Token Validator spec

hhunter-ms committed:
Merge branch 'v1.13' into actors-services

hhunter-ms committed:
Merge pull request #4002 from ItalyPaleAle/actors-services

[1.13] Document actor services flags

hhunter-ms committed:
Merge branch 'v1.13' into wfbackend-docs

hhunter-ms committed:
Merge branch 'v1.13' into issue_3868

hhunter-ms committed:
Clarify managed identities doc  (#4024)

* add more context

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* crosslink

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* Update daprdocs/content/en/developing-applications/integrations/Azure/azure-authentication/howto-mi.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

* commiting before pulling in

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* update per bernd review

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* typo and emphasis

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

---------

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

ASHIQUEMD committed:
Update daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-architecture.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: MD Ashique <noorani.ashique5@gmail.com>

ASHIQUEMD committed:
Update daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-architecture.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: MD Ashique <noorani.ashique5@gmail.com>

ASHIQUEMD committed:
Update daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: MD Ashique <noorani.ashique5@gmail.com>

ASHIQUEMD committed:
Update daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: MD Ashique <noorani.ashique5@gmail.com>

ASHIQUEMD committed:
Update daprdocs/content/en/reference/components-reference/supported-workflow-backend/actor-workflow-backend.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: MD Ashique <noorani.ashique5@gmail.com>

ASHIQUEMD committed:
Merge branch 'v1.13' into wfbackend-docs

ASHIQUEMD committed:
Update daprdocs/content/en/reference/components-reference/supported-workflow-backend/actor-workflow-backend.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: MD Ashique <noorani.ashique5@gmail.com>

ASHIQUEMD committed:
Add workflow backend documentation

Signed-off-by: ashiquemd <noorani.ashique5@gmail.com>

hhunter-ms committed:
update quickstart, start updating patterns

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
merge conflict

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3868

hhunter-ms committed:
add more pattern examples

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3868' of https://github.com/hhunter-ms/docs into issue_3868

msfussell committed:
Update daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md

Signed-off-by: Mark Fussell <markfussell@gmail.com>

hhunter-ms committed:
Merge pull request #4028 from ASHIQUEMD/wfbackend-docs

Add workflow backend documentation

hhunter-ms committed:
Merge branch 'v1.13' into issue_3868

hhunter-ms committed:
add fan in/fan out

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3868' of https://github.com/hhunter-ms/docs into issue_3868

hhunter-ms committed:
clarify support in the docs for cosmosdb

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

JoshVanL committed:
Hot reloading: add notes about the behaviour and exceptions (#4027)

* Hot reloading: add notes about the behaviour and exceptions

Moves hot reloading to a separate section in `component-updates.md`.

Adds notes about the behaviour of hot reloading regarding downtime and
respecting init errors.

Adds exceptions to the component types ignored by hot reloading.

Signed-off-by: joshvanl <me@joshvanl.dev>

* Adds link ref to workflow backends in unsupported hot reload

Signed-off-by: joshvanl <me@joshvanl.dev>

* Update component-updates.md

Edited "wills" to be present tense

Signed-off-by: Mark Fussell <markfussell@gmail.com>

---------

Signed-off-by: joshvanl <me@joshvanl.dev>
Signed-off-by: Mark Fussell <markfussell@gmail.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

hhunter-ms committed:
Update daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md

Co-authored-by: Marc Duiker <marcduiker@users.noreply.github.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3868

hhunter-ms committed:
Update daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

hhunter-ms committed:
updates per mark and marc

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3868' of https://github.com/hhunter-ms/docs into issue_3868

hhunter-ms committed:
Merge branch 'v1.13' into issue_3962

hhunter-ms committed:
Update daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md

Co-authored-by: Marc Duiker <marcduiker@users.noreply.github.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

hhunter-ms committed:
Merge pull request #3895 from hhunter-ms/issue_3868

[Workflow] Updates for Go SDK

hhunter-ms committed:
Merge branch 'v1.13' into bulk-retrieval-deserialization

hhunter-ms committed:
remove explicit callout for cosmosdb

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3962' of https://github.com/hhunter-ms/docs into issue_3962

hhunter-ms committed:
Merge pull request #3843 from WhitWaldo/bulk-retrieval-deserialization

Added example for retrieving bulk deserialized items in dotnet-sdk repo

hhunter-ms committed:
Merge branch 'v1.13' into issue_3962

hhunter-ms committed:
initial add, need to create diagram to replace placeholder

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3704

hhunter-ms committed:
Update daprdocs/content/en/operations/components/certification-lifecycle.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

hhunter-ms committed:
Update daprdocs/content/en/operations/components/certification-lifecycle.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

hhunter-ms committed:
SDK forward and backward compatibility (#4036)

* update sdk support

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* Update daprdocs/content/en/operations/support/support-release-policy.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

* Update daprdocs/content/en/operations/support/support-release-policy.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

* Update daprdocs/content/en/operations/support/support-release-policy.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

* Update daprdocs/content/en/operations/support/support-release-policy.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

---------

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3962

hhunter-ms committed:
update diagram

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3704' of https://github.com/hhunter-ms/docs into issue_3704

hhunter-ms committed:
Merge pull request #4035 from hhunter-ms/issue_3962

Clarify support for stable components using CosmosDB as example

hhunter-ms committed:
add diagram

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3704

mikeee committed:
add go limitions

Signed-off-by: mikeee <hey@mike.ee>

mikeee committed:
Merge branch 'v1.13' into wf-docs

Signed-off-by: Mike Nguyen <hey@mike.ee>

hhunter-ms committed:
[Workflow] JS quickstart docs (#4034)

* initial skeleton of quickstart doc for JS workflow

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* update with current code

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* typo, wrong order number

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

---------

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

msfussell committed:
Update sidecar-health.md

Rewrote the start of this section

msfussell committed:
Update health_api.md

Edited this PR

hhunter-ms committed:
Merge branch 'v1.13' into issue_3704

mikeee committed:
Merge branch 'v1.13' into wf-docs

artursouza committed:
Added errorIfNot2XX option to HTTP binding docs (#3741) (#4056)

* Added errorIfNot2XX option to HTTP binding docs



* Update daprdocs/content/en/reference/components-reference/supported-bindings/http.md



---------

Signed-off-by: Wim Vande Casteele <wim.vande.casteele@randstadgroup.be>
Signed-off-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Artur Souza <asouza.pro@gmail.com>
Co-authored-by: wimvc1 <86042870+wimvc1@users.noreply.github.com>
Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

mikeee committed:
Merge branch 'v1.13' into wf-docs

hhunter-ms committed:
updates per mark

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3704' of https://github.com/hhunter-ms/docs into issue_3704

hhunter-ms committed:
edit and add cross-linking

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3704

yaron2 committed:
add maxConcurrentHandlers to Pulsar (#4057)

Signed-off-by: yaron2 <schneider.yaron@live.com>

hhunter-ms committed:
Merge branch 'v1.13' into wf-docs

yaron2 committed:
Add reminders protobuf section (#4058)

* Add reminders protobuf section

Signed-off-by: yaron2 <schneider.yaron@live.com>

* Update daprdocs/content/en/developing-applications/building-blocks/actors/actors-timers-reminders.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Yaron Schneider <schneider.yaron@live.com>

* Update daprdocs/content/en/developing-applications/building-blocks/actors/actors-timers-reminders.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Yaron Schneider <schneider.yaron@live.com>

* Update daprdocs/content/en/developing-applications/building-blocks/actors/actors-timers-reminders.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Yaron Schneider <schneider.yaron@live.com>

* Update actors-timers-reminders.md

* Update actors-timers-reminders.md

typo

---------

Signed-off-by: yaron2 <schneider.yaron@live.com>
Signed-off-by: Yaron Schneider <schneider.yaron@live.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

mikeee committed:
add external interaction recommendation

Signed-off-by: mikeee <hey@mike.ee>

mikeee committed:
Merge branch 'v1.13' into wf-docs

msfussell committed:
Merge branch 'v1.13' into issue_3704

hhunter-ms committed:
Merge pull request #4052 from mikeee/wf-docs

add go limitations

hhunter-ms committed:
mark review

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3704' of https://github.com/hhunter-ms/docs into issue_3704

hhunter-ms committed:
Merge branch 'v1.13' into issue_3704

hhunter-ms committed:
resolve conflicts

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
python

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
java

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
dotnet

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
js

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
go

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
update with more slides

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3965_diagrams

hhunter-ms committed:
Merge pull request #4065 from hhunter-ms/submodules_03-04

Submodules Mar 4

hhunter-ms committed:
Merge branch 'v1.13' into upmerge_03-04

hhunter-ms committed:
Merge pull request #4064 from hhunter-ms/upmerge_03-04

Upmerge Mar 4: 1.12 --> 1.13

hhunter-ms committed:
Merge branch 'v1.13' into issue_3965_diagrams

hhunter-ms committed:
Merge pull request #3988 from hhunter-ms/issue_3965_diagrams

[Endgame] Diagram deck with 1.13 updates

hhunter-ms committed:
Merge branch 'v1.13' into issue_3704

JoshVanL committed:
Metric Cardinality: fix example setting increased cardinality. (#4062)

See https://github.com/dapr/dapr/blob/9f5a423d1809cd0d0426baf432a813886d3ff453/tests/integration/suite/daprd/metrics/httpserver_highcardinality.go#L52-L53

Signed-off-by: joshvanl <me@joshvanl.dev>
Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Co-authored-by: Yaron Schneider <schneider.yaron@live.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

msfussell committed:
Merge branch 'v1.13' into issue_3704

hhunter-ms committed:
link to js code

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'issue_3704' of https://github.com/hhunter-ms/docs into issue_3704

hhunter-ms committed:
Merge pull request #4043 from hhunter-ms/issue_3704

`/outbound` path for health checks

hhunter-ms committed:
update support doc

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
mark 1.10 as unsupported

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Update daprdocs/content/en/operations/support/support-release-policy.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

hhunter-ms committed:
Merge branch 'v1.13' into issue_3965_1.13

hhunter-ms committed:
Merge pull request #3980 from hhunter-ms/issue_3965_1.13

[Endgame] Prep v1.13 branch for release

hhunter-ms committed:
forgot website root

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
update

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge pull request #4070 from hhunter-ms/issue_3965_websiteroot

[Endgame] website-root.yml

newbe36524 committed:
Update secrets-scopes.md

Signed-off-by: Newbe36524 <newbe36524@qq.com>

newbe36524 committed:
Update gRPC-integration.md

Signed-off-by: Newbe36524 <newbe36524@qq.com>

newbe36524 committed:
Update open-telemetry-collector-jaeger.md

Signed-off-by: Newbe36524 <newbe36524@qq.com>

newbe36524 committed:
Update zipkin.md

Signed-off-by: Newbe36524 <newbe36524@qq.com>

newbe36524 committed:
Update kubernetes-job.md

Signed-off-by: Newbe36524 <newbe36524@qq.com>

newbe36524 committed:
Update middleware-bearer.md

Signed-off-by: Newbe36524 <newbe36524@qq.com>

hhunter-ms committed:
Merge pull request #4080 from newbe36524/v1.13

Remove Duplicated keys from head matter

newbe36524 committed:
add translation in zh-Hans for v1.13

Signed-off-by: newbe36524 <newbe36524@qq.com>

hhunter-ms committed:
fix copy/paste typo

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
add graceful shutdown duration section

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge pull request #4082 from newbe36524/v1.13

add translation in zh-Hans for v1.13

alicejgibbons committed:
Remove memory limits in production (#4091)

Signed-off-by: Alice Gibbons <alice@diagrid.io>

marcduiker committed:
Merge branch 'v1.13' into issue_4081

marcduiker committed:
Update table with features

Signed-off-by: Marc Duiker <marcduiker@users.noreply.github.com>

passuied committed:
Adjust documentation per `rawPayload=true` requirements (#4086)

* Adjust documentation per `rawPayload=true` requirements

Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* small tweak

Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* PR feedback

Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

* more editing

Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>

---------

Signed-off-by: Patrick Assuied <patrick.assuied@elationhealth.com>
Co-authored-by: Yaron Schneider <schneider.yaron@live.com>

tlund101 committed:
Update confusing examples for subscribePollInterval

Most of the other options show an example identical to the default value.  This was an exception and could prove confusing when reading the docs

Signed-off-by: Trevor Lund <89029310+tlund101@users.noreply.github.com>

marcduiker committed:
Merge branch 'v1.13' into update-sdk-feature-table

riosengineer committed:
Update holopin.yml (#4108)

Updated holopin.yaml file to use the default holopin badge for the docs repo.

Signed-off-by: Dan Rios <36534747+riosengineer@users.noreply.github.com>

AnnuCode committed:
adding Dapr Repository Index for contributing (#4088)

* adding Dapr Repository Index for contributing

Signed-off-by: Annu Singh <annu.singh@terramate.io>

* adding cli repo

Signed-off-by: Annu Singh <annu.singh@terramate.io>

---------

Signed-off-by: Annu Singh <annu.singh@terramate.io>
Co-authored-by: Marc Duiker <marcduiker@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

msfussell committed:
Merge branch 'v1.13' into patch-2

tlund101 committed:
Edit this page link directed to the wrong branch (#4101)

Signed-off-by: Trevor Lund <89029310+tlund101@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

hhunter-ms committed:
Add 1.13.1 and 1.13.2 to the docs (#4110)

* update support docs for patch releases

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* Update daprdocs/content/en/operations/support/support-release-policy.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

* Update daprdocs/content/en/operations/support/support-release-policy.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

---------

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

marcduiker committed:
Add Diagrid conductor to integrations (#4112)

* Add Diagrid conductor to integrations

Signed-off-by: Marc Duiker <marcduiker@users.noreply.github.com>

* Add diagram

Signed-off-by: Marc Duiker <marcduiker@users.noreply.github.com>

* Update daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md

Signed-off-by: Mark Fussell <markfussell@gmail.com>

* Update daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md

Signed-off-by: Mark Fussell <markfussell@gmail.com>

* Update daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md

Signed-off-by: Mark Fussell <markfussell@gmail.com>

* Update daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md

Signed-off-by: Mark Fussell <markfussell@gmail.com>

* Update daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md

Signed-off-by: Mark Fussell <markfussell@gmail.com>

* Update daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md

Signed-off-by: Mark Fussell <markfussell@gmail.com>

---------

Signed-off-by: Marc Duiker <marcduiker@users.noreply.github.com>
Signed-off-by: Mark Fussell <markfussell@gmail.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

msfussell committed:
Merge branch 'v1.13' into update-sdk-feature-table

marcduiker committed:
Update daprdocs/content/en/developing-applications/sdks/_index.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Marc Duiker <marcduiker@users.noreply.github.com>

marcduiker committed:
Add font usage for MacOS (#4094)

Signed-off-by: Marc Duiker <marcduiker@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

hhunter-ms committed:
Merge branch 'v1.13' into patch-2

hhunter-ms committed:
Merge branch 'v1.13' into update-sdk-feature-table

hhunter-ms committed:
Merge pull request #4102 from tlund101/patch-2

Update confusing examples for subscribePollInterval

msfussell committed:
Merge branch 'v1.13' into update-sdk-feature-table

hhunter-ms committed:
Merge pull request #4095 from marcduiker/update-sdk-feature-table

Update table with sdk features

hhunter-ms committed:
Merge branch 'v1.13' into issue_4081

hhunter-ms committed:
Merge pull request #4083 from hhunter-ms/issue_4081

Fix the copy/paste typo

hhunter-ms committed:
Merge branch 'v1.13' into issue_4063

hhunter-ms committed:
Merge pull request #4084 from hhunter-ms/issue_4063

Add section on graceful shutdown duration

james-bartlett committed:
Clarify that disabling kubernetes built in secret store can only be done with an annotation, not by setting a helm argument (#4124)

Clarify that disabling kubernetes built in secret store can only be done with an annotation, not by setting a helm argument

Signed-off-by: james-bartlett <jamesmbartlett@outlook.com>

hhunter-ms commented:
@WhitWaldo friendly bump

hhunter-ms committed:
adding missing queueName (#4125)

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Co-authored-by: Yaron Schneider <schneider.yaron@live.com>

WhitWaldo commented:
@hhunter-ms Forgot all about this - I'll see if I can't get something working soon. Thanks!

WhitWaldo committed:
Merge branch 'dapr:v1.13' into limit-workflow-concurrency-dotnet

WhitWaldo committed:
Updatex example, modifying sample at https://stackoverflow.com/questions/58789750/controlling-the-max-degree-of-parallelism-in-the-fan-out-fan-in-pattern-in-durab/58794510#58794510 to include support for returning tasks with results

Signed-off-by: Whit Waldo <whit.waldo@innovian.net>

WhitWaldo commented:
@hhunter-ms Updated the docs to reflect the example given in the SO post to include support for retaining a returned value for each of the tasks

hhunter-ms commented:
@WhitWaldo it looks like 1.13 was pulled into this PR, which is pointing into 1.12 - do you mind either reopening as a new PR, or somehow reverting the 1.13 merge?
